DNN Platform/Library/Entities/IPFilter/IPFilterController.cs

```diff
@@ -256,5 +261,20 @@ public bool IsAllowableDeny(string currentIP, IPFilterInfo ipFilter)
         }
 
         #endregion
+
+        private static void AssertValidIPFilter(IPFilterInfo ipFilter)
+        {
+            IPAddress parsed;
+            if (IPAddress.TryParse(ipFilter.IPAddress, out parsed) == false)
+            {
+                throw new ArgumentException("IP address is not in correct format");
```

Comment:
What is the best practice for adding localization? Maybe we don't want to localize this message at all?